docs/geedocs/5.2.2/index.html

```diff
@@ -20,10 +20,10 @@
   <div class="sidebar1"> </div>
   <div class="content"> <a name="top_of_file"></a>
     <p><img src="art/common/googlelogo_color_260x88dp.png" width="130" height="44" alt="Google logo" /></p>
-    <h2>Google Earth Enterprise Documentation Version 5.2.1</h2>
+    <h2>Google Earth Enterprise Documentation Version 5.2.2</h2>
     <p>This documentation contains legacy information that may not apply to the
```

Comment:
Is this information is up to date ? Should we keep that paragraph as it is or should we simply remove it ?

Reply:
This task was really about updating the versions. A more in-depth review should probably probably be a follow-up. Maybe attached to the release notes task? But, if you have a specific suggestion for a change, I'll make it.

Reply:
We could have a follow up ticket for it...or my suggestion would be simply remove that text as it does not seem to apply any more (as we are now made two open sourced releases).